tools/create_data.py

```diff
@@ -198,6 +199,21 @@ def waymo_data_prep(root_path,
         num_worker=workers).create()
 
 
+def once_data_prep(root_path, info_prefix = 'once', split = 'trainval'):
```

Comment:
We'll appreciate it if you can describe the consuming time of this dataset conversion and the quota of dataset storage in the PR message.